packages/ui/client/components/views/ViewModuleGraph.vue

```diff
@@ -55,15 +72,17 @@ function resetGraphController() {
           radiusMultiplier: 2,
         },
       },
+      // TODO: pointerup triggers when drag ends
+      // modifiers: {
+      //   node(selection: Selection<SVGCircleElement, ModuleNode, SVGGElement, undefined>) {
+      //     selection
+      //       .on('pointerdown', null)
+      //       .on('pointerup', (_: PointerEvent, node: ModuleNode) => setSelectedModule(node.id))
+      //   },
+      // },

```

Comment:
@antfu This part is relevant for you, as you can access the d3 selection directly now.
Just keep in mind that `pointerup` also triggers when a drag ends, so some logic would have to be implemented.

I updated the old callback to only fire on double-clicks (or double-taps), so this works in the meantime.